source/JsMaterialX/README.md

```diff
@@ -1,14 +1,18 @@
-# Javascript Support
```

Comment:
Just some minor changes up here that shouldn't lead to conflicts with @aGuluzade's work. I would like to refactor the building section once the Docker changes are merged.